python/strings_udf/strings_udf/__init__.py

```diff
@@ -87,6 +88,29 @@ def _get_ptx_file():
         return regular_result[1]
 
 
+# Maximum size of a string column is 2gb
+STRINGS_UDF_DEFAULT_HEAP_SIZE = 2**31
+STRINGS_UDF_HEAP_SIZE = os.environ.get(
+    "STRINGS_UDF_HEAP_SIZE", STRINGS_UDF_DEFAULT_HEAP_SIZE
+)
+heap_size = 0
+
+
+def set_malloc_heap_size(size=STRINGS_UDF_HEAP_SIZE):
```

Comment:
This default parameter feels sort of weird. If (for whatever reason) you go in later and say `STRINGS_UDF_HEAP_SIZE = some_random_number` it won't change the default parameter for this function, which is encoded at definition time. Is that what you intend/do we care?